lib/magma/actions/update_attribute.rb

```diff
@@ -9,12 +9,8 @@ def initialize(project_name, action_params = {})
     end
 
     def perform
-      @action_params.slice(*Magma::Attribute::EDITABLE_OPTIONS).each do |option, value|
-        attribute.update_option(option, value)
-      rescue => e
-        @errors << Magma::ActionError.new(message: 'Update attribute failed', source: @action_params.slice(:attribute_name, :model_name), reason: e)
-      end
-      return @errors.empty?
+      attribute.update(@action_params.slice(*Magma::Attribute::EDITABLE_OPTIONS))
+      @errors.empty?
```

Comment:
It's not clear to me now how @errors gets populated

Reply:
`@errors` will get set when call `validate` on an action.

https://github.com/mountetna/magma/blob/ef767145c933f0186474a3efce27b7170693be8e/lib/magma/server/update_model.rb#L11

https://github.com/mountetna/magma/blob/ef767145c933f0186474a3efce27b7170693be8e/lib/magma/actions/update_attribute.rb#L31-L44

Before `#perform` could also populate `@errors` if an error happened during the attribute update. Now that we're using `Sequel::Model#update` to update attributes, exceptions are a lot less likely to happen. I tried to make it error but was unsuccessful.

We _probably_ don't need to return `@errors.empty?` from `#perform` anymore - it just needs to return a truthy value.